backend/src/ml_space_lambda/utils/iam_manager.py

```diff
@@ -192,6 +192,7 @@ def __init__(self, iam_client=None, sts_client=None):
         env_variables = get_environment_variables()
         self.data_bucket = env_variables["DATA_BUCKET"]
         self.system_tag = env_variables["SYSTEM_TAG"]
+        self.app_role_name = os.getenv("APP_ROLE_NAME", "")
```

Comment:
Should we just add `APP_ROLE_NAME` to the environment variables so it can be read from `env_variables` too? 

Also if we have the enum for environment variables shouldn't we start using them in all the files we touch?

Reply:
Yes. I scaled this back though because I think `get_environment_variables` was a culprit for mocking issues and I was trying to get around it so I could proceed. I am all for updating

I will try it, if it causes test failures I will let you know and we can revisit it in a separate refactor